hashring/hashring.go

```diff
@@ -64,7 +64,7 @@ func (r *HashRing) emit(event interface{}) {
 	}
 }
 
-// RegisterListener adds a listener that will be sent swim events.
+// RegisterListener adds a listener that will listens for HashRing events.
```

Comment:
Not sure about this. I actually think this comment was more accurate before. In any case, there is a typo in the updated comment.